### PR TITLE
Update readme to emphasise fact that no ServerQuery account is required

### PR DESCRIPTION
**Pull request by djmattyg007**
It took me quite a while to work out that I didn't have to supply any
ServerQuery credentials, so I figured a readme update was in order, to
hopefully reduce this confusion for others.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,20 +11,21 @@ A simple Teamspeak stat-generator - based on server-logs
...
-Parse logs matching `ts3server*.log` in `/var/log/teamspeak3-server` and write output to `/var/www/tsstats.html`
+Parse logs matching `ts3server*.log` in `/var/log/teamspeak3-server` and write output to `/var/www/tsstats.html`.
 
 # CLI-Usage
 ```
 usage: tsstats [-h] [-c CONFIG] [--idmap IDMAP] [-l LOG] [-o OUTPUT] [-d]
                    [-nod]
 
-A simple Teamspeak stats-generator - based on server-logs
+A simple Teamspeak stats-generator - based solely on server-logs
```
Comment (Thor77) [inline README.md]: Please change that [here](https://github.com/Thor77/TeamspeakStats/blob/master/tsstats/__main__.py#L18) instead, as I would accidentially overwrite it this way, once I add smth new to the cli (as that's just the output of `tsstats -h`).

Comment (djmattyg007) [inline README.md]: Thanks. Should I leave this change here, or remove it?

Comment (Thor77) [inline README.md]: You can just leave it there

**Comment by Thor77** [comment]
Thank you very much 👍

**Pull request merged**